downstream/modules/hub/proc-uploading-collections.adoc

```diff
@@ -14,6 +14,12 @@ Format your collection file name as follows: <my_namespace-my_collection-1.0.0.t
 .Prerequisites
 * You have a namespace to which you can upload the collection.
 
+[IMPORTANT]
+
+====
+{PlatformNameShort} does not support uploading collections larger than 2GB. Uploading collections larger than 2GB will result in an error. In scenarios that require a complete environment with multiple collections and dependencies, use an {ExecEnvShort}. See link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.6/html/managing_automation_content/managing-containers-hub#obtain-images[Pulling {ExecEnvShort}s for use in {HubName}] for more information.
```

Comment:
I approved, however, I think stating 2GB here may be read as a suggestion or endorsement of having a 2GB collection.

We want to have collections really smaller than that, not only for uploading limits but to avoid headaches on import/export, linting, auditing, etc.

Can you add in some way that we recommend that collections preferably will have around 250MB in size.